backend/dataall/modules/dataset_sharing/db/share_object_repositories.py

```diff
@@ -849,17 +849,20 @@ def _find_all_share_item(session, share, status, share_type_model, share_type_ur
         )
 
     @staticmethod
-    def find_all_share_items(session, share_uri, share_type):
-        return (
+    def find_all_share_items(session, share_uri, share_type, status=None):
```

Comment:
why do we pass this `status` parameter?

Reply:
To check the share item status, now we need to filter by that field. For example for "delete table permissions to requesters for revoked tables" we need to find all share items that are tables and are in Revoke_Approved